modules/apps/frontend-js/frontend-js-collapse-support-web/src/main/resources/META-INF/resources/CollapseProvider.ts

```diff
@@ -116,7 +116,7 @@ class CollapseProvider {
 			});
 
 			panel.classList.add(CssClass.COLLAPSING);
-			panel.style[dimension] = 0;
+			panel.style[dimension] = '';
```

Comment:
This seems to fix the issue, but we are setting a string value to something that should be a number. This may have weird effects in some older or obscure browsers. The goal is to have the style set to an unset value, right? Can we do it directly, something like:
```js
panel.style.removeProperty(dimension)
```

Reply:
Looks like `removeProperty` also resolves the issue! I've applied the update.

Reply:
@holatuwol @markocikos @sontruongces 
it seems the issue was not fully covered (see the comment from CSE: https://issues.liferay.com/browse/LPP-45655?focusedCommentId=2724103&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2724103), there are still two lines at 166 & 174 where the style is setting a string value to something that should be a number. IMO it should be consistent in all places with _removeProperty_
https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-collapse-support-web/src/main/resources/META-INF/resources/CollapseProvider.ts